services/QuillLMS/babel.config.cjs

```diff
@@ -0,0 +1,6 @@
+module.exports = {
```

Comment:
babel.config.cjs is not preferred over .babelrc

Source: https://github.com/jestjs/jest/commit/1818d84b2cf20ecb8db2215b3ad6b5e397ab2673#diff-04c6e90faac2675aa89e2176d2eec7d8